README.md

```diff
@@ -47,7 +47,7 @@ be added to the project using the `--feature` options, this can be used multiple
 
 ## GUI version
 
-The GUI version of the tool, run by adding `--gui` to the command line, uses `tkinter` to provide a platform agnostic script that will run on Linux, Mac and Windows. All the options from the command line tool are also supported in the GUI.
+The GUI version of the tool, run by adding `--gui` to the command line, uses `tkinter` to provide a platform agnostic script that will run on Linux, Mac and Windows. All the options from the command line tool are also supported in the GUI. Install `python3-tk` package for GUI support on Ubuntu/Debian/RPI platforms.
```

Comment:
Perhaps "It may be necessary to install the ..." rather than "Install ..." ?  (Based on James' comments)

Reply:
Absolutely, I will make the correction and do the final test on the RPI3 I have.